### DISCUSSION TITLE
storage/hwameistor/install/prereq/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1681] [Announcements]
# storage/hwameistor/install/prereq/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/storage/hwameistor/install/prereq/

**Comment by liyuerich** [comment]
https://hwameistor.io/cn/docs/quick_start/install/prereq
需要和这个文档一致吗？
